pre-docs/running_bolt_from_docker_image.md

```diff
@@ -0,0 +1,182 @@
+# Puppet-bolt container
+
+Puppet Bolt is available as a docker image. Container images are published to [Docker Hub](https://hub.docker.com/r/puppet/puppet-bolt/tags) with tags corresponding to bolt system package and Rubygem versions in addition to a `latest` tag which points at the newest version. This document covers some possible ways to use the container.
```

Comment:
Maybe there's a packaging nuance here I'm unaware of, but I think it should be `Bolt system package` (capitalized)